runtime/doc/news.txt

```diff
@@ -210,6 +210,9 @@ These existing features changed their behavior.
   more emoji characters than before, including those encoded with multiple
   emoji codepoints combined with ZWJ (zero width joiner) codepoints.
 
+• |vim.tbl_deep_extend()| no longer ignores any values that return `true`
+  from |vim.isarray()|.
```

Comment:
```suggestion
• |vim.tbl_deep_extend()| no longer ignores any values for which |vim.isarray()| returns `true`.
```

(Sorry, only realized afterwards that it was the inverted grammar that bugged me.)